kubernetes/services/api-gateway.yaml

```diff
@@ -4,23 +4,19 @@ metadata:
   name: api-configmap
 data:
   application.properties: |
+    ignite.thin.client.enable=true
```

Comment:
Let us move the config back to where it was, along with other Ignite configs.

Reply:
Done